docs/src/pages/StatusPage/StatusPage.js

```diff
@@ -28,6 +28,10 @@ import webStatus from './content/webStatus.md';
 import nativeStatus from './content/nativeStatus.md';
 import iosStatus from './content/iosStatus.md';
 import androidStatus from './content/androidStatus.md';
+import ecsStatus from './content/ecsStatus.md';
```

Comment:
Can these files be renamed to be more clear/meaningful? Makes maintaining easier 🙂

Reply:
![absolutely](https://media2.giphy.com/media/3olNjWU2Dxm1iNzwIh/giphy.gif)